### PR TITLE
Add df.md

### PR DESCRIPTION
**Pull request by yijunhou**
# Advanced HW 7 submission

Closes #263 

Name: Yijun
Uniqname: yijunhou

### CODE REVIEW

**Comment by mterwill** [comment]
Could you please fill out the PR template?

File: _commands/basics/df.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+---
+
+df
+--
+
+df displays the amount of disk space available on the file system containing each file name argument. If no file name is given, the space available on all currently mounted file systems is shown. Disk space is shown in 1K blocks by default, unless the environment variable POSIXLY_CORRECT is set, in which case 512-byte blocks are used.
```
Comment (mterwill) [inline _commands/basics/df.md]: This is straight out of the manpage – the goal for the reference pages is to be a little friendlier. Could you trim this down to only a sentence or two about `df` that's more elementary?

**Comment by yijunhou** [comment]
Filled. Thanks! @mterwill

File: _commands/basics/df.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+tmpfs             204848   21672    183176  11% /run
+/dev/sda1       49702628 5077532  42077232  11% /
+...(Some more info)
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `df -h`
```
Comment (mterwill) [inline _commands/basics/df.md]: I might even consider using this as the primary example... I always use `df -h`.

File: _commands/basics/df.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+~~~ bash
+$ df Desktop/
+Filesystem     1K-blocks    Used Available Use% Mounted on
+/dev/sda1       49702628 5077556  42077208  11% /
+~~~
+
+#### `-a, --all`
+~~~ bash
+include dummy file systems.
```
Comment (mterwill) [inline _commands/basics/df.md]: What's a dummy filesystem? If it's not a useful option, feel free to take it out.

File: _commands/basics/df.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+~~~
+
+#### `--total`
+~~~ bash
+display a grand total.
+~~~
+
+#### `-i, --inodes`
+~~~ bash
+list inode information instead of block usage.
```
Comment (mterwill) [inline _commands/basics/df.md]: Again, unless you explain what an inode is, I wouldn't bother putting this here.

**Comment by yijunhou** [comment]
@mterwill Revised a little bit to make it more concise.

**Comment by mterwill** [comment]
Great! Thanks. Merging + giving you credit.

**Pull request merged**